src/prowjobsscraper/indices/usages_schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "settings": {
+      "index": {
+        "number_of_shards": "1",
+        "number_of_replicas": "0",
```

Comment:
just out of curiosity - why 0 replicas?

Reply:
I followed the other sachems, I am not sure. @adriengentil ?

Reply:
I guess it just happened to be 0 :D
@rccrdpccl how many replicas would you recommend? They are all small indices.

Reply:
0 replicas is vulnerable for data loss if something goes wrong in the node where the shard is assigned.
As it's small and we have no concerns of filling up the disk, I'd recommend 2 replicas (plus primary).
With the caveat that in integration we only have one node so it would need to be 0 there, otherwise it be marked as "yellow".

In any case, it's not important as we also have backups :)